src/wallet.ts

```diff
@@ -379,7 +382,7 @@ export function createWalletMiddleware({
   ): Promise<string> {
     if (typeof address === 'string' && address.length > 0) {
       // ensure address is included in provided accounts
-      const accounts: string[] = await getAccounts(req);
+      const accounts: string[] = await getAccounts(req, false);
```

Comment:
We should add a comment explaining what the `false` is doing here, I think.

Reply:
A comment was added, but we'll still get the misleading error on L395 below if the address is valid and it's not contained in the array returned from `getAccounts`.